cli/src/lib/makeBundle.js

```diff
@@ -30,7 +30,31 @@ module.exports = (inDir, outFile) => {
         })
 
         archive.pipe(output)
-        archive.directory(inDir, false)
+
+        // avoid packing the created file stream
```

Comment:
Just out of curiosity, what does this mean exactly?

Reply:
It means that if we just pass in the `.directory(inDir)` and create the file stream in `inDir/bundle.zip`, then the `bundle.zip` is going to end up inside the zip (https://github.com/archiverjs/node-archiver/issues/319).

This is not much of a problem with the defaults for an app, but for a lib it can be.